### ISSUE TITLE
Missing alt text

### ISSUE THREAD

**Issue by sstevens2** [UW-Madison-DataScience/github-skillup-2023-09-11#4]
There is no alt text for the image in the recipe instructions.

I recommend...
A bunch of cinnamon rolls in a baking tray in an oven.

**Comment by sstevens2** [comment]
I'll go ahead and make a PR with this change.
